infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -403,10 +391,6 @@ private boolean runRecoveryReconfiguration() {
         boolean recoveryResult = clusterLayout.getEpoch() > localRecoveryLayout.getEpoch()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "recoveryResult". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)